install/ui/src/freeipa/widget.js

```diff
@@ -3528,8 +3528,12 @@ IPA.column = function (spec) {
 
         var c;
         if (that.link && !suppress_link) {
+            var entity_hash = navigation.get_entity_hash(
+                that.entity.name, 'details', [value]
```

Comment:
Fixed.

I added `get_entity_path` method to column object and overrode it for some cases. It is used by default link_handler implementation.

In case link_handler is overridden, `get_entity_path` would be called anyway in the part of code, but it would return empty hash without any side-effects or errors.